terraform/projects/app-ecs-services/config/alerts/data-gov-uk.yml

```diff
@@ -32,3 +32,11 @@ groups:
     annotations:
         summary: "Index size of Elasticsearch for {{ $labels.job }} has decreased significantly"
         description: "The index size of Elasticsearch for {{ $labels.job }} has decreased by more than 100 documents in the last 30 minutes"
+  - alert: DataGovUk_HighSidekiqEnqueuedJobs
+    expr: sidekiq_enqueued_jobs > 1000
```

Comment:
I think so.
@idavidmcdonald - we did chat about this in the lab on Friday, but I can't remember - can we target only production?

Reply:
You can indeed only target production by using `sidekiq_enqueued_jobs{job="publish-data-production-queue-monitor"}` although please do check this in the prometheus interface to double check this gives you what you want

Reply:
I think we should add something to this query to make it specific to DGU in any case: otherwise, if some other user adds sidekiq metrics from their own services, this alert might alert on that other metric.  Probably on the `org` or `space` label?

Good work capturing that 1000 is a guess in the commit message 👍 this will make it easier for people to make an informed decision if they want to change it later.

Reply:
@idavidmcdonald  - thanks - that worked!

@thomasleese I've updated the query now so it should only pull through prod 5cb74b3

@philandstuff - thanks for the hint - I've added in the org label to the query - also added a grafana screenshot to support my guess :)